st2api/st2api/config.py

```diff
@@ -28,10 +28,23 @@
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
+def parse_args(args=None):
+    CONF(args=args, version=VERSION_STRING)
+
+
+def register_opts():
+    _register_common_opts()
+    _register_app_opts()
+
+
 def _register_common_opts():
     common_config.register_opts()
 
 
+def get_logging_config_path(service='api'):
```

Comment:
Ahh. I understand now. Perhaps a better refactoring would be to move the notifier out of st2actions at some point. But yeah, having packages inside packages is fine. Not sure how this would affect packaging though. Gotta be careful not to break stuff. I'd do this in another PR just to be safe.